src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -466,7 +466,6 @@ public static void DropDatabase(SqlConnection sqlConnection, string dbName)
 
         public static bool IsLocalDBInstalled() => !string.IsNullOrEmpty(LocalDbAppName?.Trim()) && IsIntegratedSecuritySetup();
         public static bool IsLocalDbSharedInstanceSetup() => !string.IsNullOrEmpty(LocalDbSharedInstanceName?.Trim()) && IsIntegratedSecuritySetup();
-
```

Comment:
Unnecessary change! Could be reverted.

Reply:
still not reverted!

Reply:
Weird, Can you make new line suggestion please, not sure by new line did not mark it outdated.